mission_control/navigator_alarm/navigator_alarm/alarm_handlers/network_loss.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import rospy
+from _template import HandlerBase
+
+from lqrrt_ros.msg import MoveAction, MoveGoal
```

Comment:
I was going to have network kill drive the boat back to a safe spot, which would need these (never got around to it though)